lib/Serialization/DeclTypeRecordNodes.def

```diff
@@ -134,7 +134,7 @@ DECL(PRECEDENCE_GROUP)
 DECL(ACCESSOR)
 
 #ifndef DECL_ATTR
-#define DECL_ATTR(NAME, CLASS, OPTIONS, CODE) RECORD_VAL(CLASS##_DECL_ATTR, 100+CODE)
+#define DECL_ATTR(NAME, CLASS, OPTIONS, CODE) RECORD_VAL(CLASS##_DECL_ATTR, 90+CODE)
```

Comment:
I am not proud of this, but there are 99 existing decl attrs, so with one more we would otherwise crash into `FIRST_PATTERN(PAREN, 200)`.